File: JavaScript/Exercicios - FACCAT/Exercicio-28/exercicio28.js
// Exercícios, Lógica de Programação, Algoritmos e JavaScript
// Ler 3 valores (considere que não serão informados valores iguais) e escrever o maior deles

// Campo do exercício 28
let numero1, numero2, numero3; //variável

alert("Ler 3 valores e escrever o maior deles") //saída de dados

numero1=parseInt(prompt("Digite um número: ")); //entrada e saída de dados
numero2=parseInt(prompt("Digite outro número: ")); //entrada e saída de dados
numero3=parseInt(prompt("Digite mais um número: ")); //entrada e saída de dados

if ((numero1 == numero2) || (numero1 == numero3) || (numero2 == numero3)) {
    alert("Digite números diferentes");
}else if((numero1 > numero2) && (numero1 > numero3)){
    alert("O número maior é: "+numero1);
}else if ((numero2 > numero1) && (numero2 > numero3)){
    alert("O número maior é: "+numero2);
}else {
    alert("O número maior é: "+numero3);
};

//Processamento

 //saída de dados 

File: Java/Exercicios - FACCAT/Exercicio28.java
package javaBasico;

import java.util.Scanner;

public class Exercicio28 {
	
	public static void main(String[] args) {
		
		Scanner dados = new Scanner(System.in);
		
		System.out.println("Ler 3 valores e escrever o maior deles");
		
		System.out.println("Digite um número: ");
		int numero1 = dados.nextInt();
		
		System.out.println("Digite outro número: ");
		int numero2 = dados.nextInt();
		
		System.out.println("Digite mais um número: ");
		int numero3 = dados.nextInt();
		
		if ((numero1 == numero2) || (numero1 == numero3) || (numero2 == numero3)) {
			System.out.println("Digite número diferentes!");
			
		}else if ((numero1 > numero2) && (numero1 > numero3)) {
			System.out.println("O número maior é: "+numero1);
			
		}else if ((numero2 > numero1) && (numero2 > numero3)) {		
			System.out.println("O número maior é: "+numero2);
			
		}else {
			System.out.println("O número maior é: "+numero3);
		}
		
		dados.close();
	}
}

// Exercícios, Lógica de Programação, Algoritmos e JavaScript

/* Ler 3 valores (considere que não serão informados valores iguais) e escrever o maior deles. */
